README.md

```diff
@@ -86,6 +86,12 @@ WARNING: Use these accounts and their keys ONLY for local testing. DO NOT use th
 
 Compile Cairo contracts. Compilation artifacts are written into the `artifacts/` directory.
 
+```txt
+nile compile [PATH_TO_CONTRACT | --directory DIRECTORY] [--disable-hint-validation]
```

Comment:
```suggestion
nile compile [PATH_TO_CONTRACT] [--directory DIRECTORY] [--disable-hint-validation]
```
`PATH_TO_CONTRACT` and `--directory` are not really alternatives aren't they?

Reply:
To my eyes, when you add `PATH_TO_CONTRACT`, `--directory` serves no purpose and vice versa, no?

`nile compile path/to/contract.cairo`

and

`nile compile --directory path/to/directory`

Reply:
I believe it also sets the cairo_path for the compiler. for example, this is needed to compile OZ contracts:
```sh
nile compile src/openzeppelin/account/presets/EthAccount.cairo --directory src
```